src/components/core/geoChart/tests/GeoChartLegendRenderer.spec.tsx

```diff
@@ -0,0 +1,156 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { mount } from "enzyme";
+import { Execution } from "@gooddata/typings";
+import { withIntl } from "../../../visualizations/utils/intlUtils";
+import GeoChartLegendRenderer, { IGeoChartLegendRendererProps } from "../GeoChartLegendRenderer";
+import PushpinSizeLegend from "../legends/PushpinSizeLegend";
+import { IGeoConfig } from "../../../../interfaces/GeoChart";
+
+function getExecutionResponse(): Execution.IExecutionResponses {
+    const executionResult: Execution.IExecutionResult = {
```

Comment:
If possible, please reuse the existing function `getExecutionResult` from 
`stories/data/geoChart.ts`

And create new functions for `executionResponse`, `mdObject`.

Reply:
We already have one in my PR
I already draft one here
https://github.com/gooddata/gooddata-react-components/pull/1315/files#diff-81c9a5878679f7dd5e0f74ba1a410ff6R1100